docs/getting-started.md

```diff
@@ -28,8 +46,8 @@ $ SCALARDB_HOME=/path/to/scalardb
 $ cd $SCALARDB_HOME
 $ ./gradlew installDist
 $ sudo mkdir /var/log/scalar; sudo chmod 777 /var/log/scalar
-$ cd tools/schema
-$ make
+$ cd tools/scalar-schema
+$ lein uberjar
```

Comment:
I don't think it is user-friendly to make users build the jar in the getting started document.
Since it is jar, we should upload the jar to GitHub and just use it.
Thought?

Reply:
Thank you for the review comments.
jar file added into schema-tool and the document updated accordingly.